### PR TITLE
feat: add charts for a self-hosted signet setup

### PR DESCRIPTION
**Pull request by openoms**
Discussed in: https://github.com/GaloyMoney/charts/issues/1123

Depends on: https://github.com/GaloyMoney/galoy/pull/1385 (merged)

### CODE REVIEW

**Comment by openoms** [comment]
The exporter pod is failing:
```
exporter-5dbcc87b85-852bp                      0/1     CrashLoopBackOff   17 (106s ago)   70m
k3d@debian:~$ k -n galoy-dev-galoy logs exporter-5dbcc87b85-852bp
Defaulted container "exporter" out of: exporter, wait-for-mongodb-migrate (init)
/app/node_modules/bitcoin-core/dist/src/index.js:71
      throw new Error(`Invalid network name "${network}"`, {
      ^

Error: Invalid network name "signet"
    at new Client (/app/node_modules/bitcoin-core/dist/src/index.js:71:13)
    at new BitcoindClient (/app/lib/services/bitcoind/index.js:20:23)
    at Object.<anonymous> (/app/lib/services/bitcoind/index.js:87:33)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at Module.Hook._require.Module.require (/app/node_modules/require-in-the-middle/index.js:80:39)
    at require (node:internal/modules/cjs/helpers:102:18)
```

**Comment by openoms** [comment]
Both the regtest and signet setup is functional locally and the Makefile is extended with new entries.

To test the signet deployment:
```
cd charts/dev
git checkout dev-signet
make all-signet
```

**Review by krtk6160 — COMMENTED** [review]
Just have a comment regarding the docs, otherwise LGTM

File: dev/README.md [hunk 1]
```diff
@@ -19,14 +20,97 @@ make deploy-services
 make deploy
 ```
 
-Test if its working:
+### Test
+
+#### Public endpoint
```
[the remark this answers is not in the document]
Comment (openoms) [reply dev/README.md:25]: yes, it was only useful when debugging nginx, can remove.

The good thing is the public endpoint test is the same what the smoketest calls.

[the remark this answers is not in the document]
Comment (openoms) [reply dev/README.md:25]: The reason to put the test for the price was that it does work (same as in the smoketest).
Found this unrelated to the changes in this PR the authentication with twilio does not work (as noted here https://github.com/GaloyMoney/charts/pull/1685):

with the port 80 forwarded:
```
kubectl -n galoy-dev-ingress port-forward svc/ingress-nginx-controller 8080:80

curl 'localhost:8080/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($in
put: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+59981730222","code":"111111"}}}'                                       
<html>                                                                                                                                                                   
<head><title>308 Permanent Redirect</title></head>                                                                                                                       
<body>                                                                                                                                                                   
<center><h1>308 Permanent Redirect</h1></center>                                                                                                                         
<hr><center>nginx</center>                                                                                                                                               
</body>                                                                                                                                                                  
</html>                         
```

with the https port forwarded:
```
kubectl -n galoy-dev-ingress port-forward svc/ingress-nginx-controller 8080:443

curl -k 'https://localhost:8080/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($input: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+59981730222","code":"111111"}}}'
{"errors":[{"message":"missing key for twilio","locations":[{"line":1,"column":43}],"path":["userLogin"],"code":"INTERNAL_SERVER_ERROR"}],"data":null}
```

I think debugging this should be done in a separate change after this is merged so testing on signet can carry on.

[the remark this answers is not in the document]
Comment (openoms) [reply dev/README.md]: ok, sorry for the long-winded comment. Fixed twilio and removed usin the 4002 port from the readme in [959c5da](https://github.com/GaloyMoney/charts/pull/1125/commits/959c5da8cf44a104e1d86caa1b01caa6a8aeeb77)

**Review by krtk6160 — APPROVED** [review]

**Pull request merged**